Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -279,6 +279,22 @@ open class ChatChannelListVC: _ViewController,
         )
     }
     
+    open func controller(_ controller: ChatChannelListController, shouldAddNewChannelToList channel: ChatChannel) -> Bool {
+        // This is an ugly hack to call default protocol implementation
+        // see https://stackoverflow.com/a/32606899
+        class Dummy: ChatChannelListControllerDelegate {}
+        let dummy = Dummy()
+        return dummy.controller(controller, shouldListUpdatedChannel: channel)
+    }
+    
+    open func controller(_ controller: ChatChannelListController, shouldListUpdatedChannel channel: ChatChannel) -> Bool {
+        // This is an ugly hack to call default protocol implementation
+        // see https://stackoverflow.com/a/32606899
+        class Dummy: ChatChannelListControllerDelegate {}
+        let dummy = Dummy()
+        return dummy.controller(controller, shouldListUpdatedChannel: channel)
+    }
```

Comment:
Can't we simply remove the default implementation? And add it to the `ChatChannelListVC` and also remove the `AnyChannelListControllerDelegate` ?

Reply:
makes a lot sense to me

Reply:
working on it, will update the PR shortly